practice-app/server/src/app.js

```diff
@@ -4,11 +4,11 @@ const morganBody = require('morgan-body');
 const { api } = require("./routes/api");
 const app = express();
 
-// app.use(
-//   cors({
-//     origin: process.env.FRONTEND_URL,
-//   })
-// );
+app.use(
+  cors({
+    origin: process.env.FRONTEND_URL,
```

Comment:
Can you create a .env.example file so that everyone can see and provide necessary env variables?

Reply:
https://github.com/bounswe/bounswe2022group3/blob/master/practice-app/server/README.md#env

Reply:
I think it would be better to have a file instead of blocks in ReadME files since everyone could easily add to the example file. We will have many API calls and API Tokens hence having them in readmes would be cumbersome. We can discuss about this in the next team meeting if you want.

Reply:
added .env.example